modules/assemble_taxa.nf

```diff
@@ -42,7 +53,62 @@ process extract_reads {
 // either be as a result of the pipeline failing for one of many reasons, or because the file generated at the end
 // of a successful pipeline run is empty, in which case it would be meaningless to copy
 // I've not had a successful test case with the very small numbers of reads/database on my laptop
+
+process flye_assemble {
```

Comment:
I haven't tested either of these but if we choose to do DN assemblies these containers should work